### ISSUE TITLE
PHP Fatal error:  Declaration ... must be compatible with AbstractProcessingHandler

### ISSUE THREAD

**Issue by clxmstaab** [lesstif/php-jira-rest-client#386]
since I have disabled the log file recently

```
        return new ArrayConfiguration(
            [
                'jiraLogEnabled' => false,
                'jiraHost' => self::HOST,
                'jiraUser' => JiraCredentials::API_USER,
                'jiraPassword' => JiraCredentials::API_PASSWORD,
            ]
        );
```

I am getting signature fatal errors with 1.45.0 

```
PHP Fatal error:  Declaration of JiraRestApi\NoOperationMonologHandler::write(array $record) must be compatible with Monolog\Handler\AbstractProcessingHandler::write(array $record): void in /deploy/logitel/fast-track/clxlogitel/test/vendor/lesstif/php-jira-rest-client/src/NoOperationMonologHandler.php on line 16

```

**Comment by clxmstaab** [comment]
it seems the cause of this [problem was fixed with 2.0](https://github.com/lesstif/php-jira-rest-client/commit/566571cd545976b965819fa89f4ed66e5c47a0c9), but wasn't in the 1.x version

**Comment by lesstif** [comment]
hi @clxmstaab 

the 1.x version is deprecated, could you update to 2.x version please?
